src/hooks/useNewTarget.js

```diff
@@ -0,0 +1,5 @@
+import { useSelector, shallowEqual } from 'react-redux';
+
+const useNewTarget = () => useSelector(state => state.targetReducer, shallowEqual);
```

Comment:
This could also return the setter function that updates this value on redux (currently on NewTarget component)